templates/identity/email-password/register-response.html.ejs

```diff
@@ -13,30 +13,45 @@
   </header>
   <main>
     <h1>You are signed up</h1>
+    <p>
+      <strong>Welcome to Solid.</strong>
+      We wish you an exciting experience!
+    </p>
+
+    <% if (createPod) { %>
+      <h2>Your new pod</h2>
+      <p>
+        Your new pod is located at <a href="<%= podBaseUrl %>" class="link"><%= podBaseUrl %></a>.
+        <br>
+        You can store your documents and data there.
+      </p>
+    <% } %>
+
     <% if (createWebId) { %>
+      <h2>Your new WebID</h2>
       <p>
         Your new WebID is <a href="<%= webId %>" class="link"><%= webId %></a>.
+        <br>
+        You can use this identifier to interact with Solid pods and apps.
       </p>
     <% } %>
+
     <% if (register) { %>
+      <h2>Your new login</h2>
       <p>
-        You can now identify as <a href="<%= webId %>" class="link"><%= webId %></a>
-        on this server using <em><%= email %></em>.
+        You can <a href="./login">log in</a>
+        with your WebID <a href="<%= webId %>" class="link"><%= webId %></a>
+        on this server via your email address <em><%= email %></em>.
       </p>
       <% if (!createWebId) { %>
         <p>
-          Make sure you add the triple
+          Please add the triple
           <code><%= `<${webId}> <http://www.w3.org/ns/solid/terms#oidcIssuer> <${oidcIssuer}>.`%></code>
-          to your WebID profile.
+          to your existing WebID document <em><%= webId %></em>
+          to indicate that you trust this server to act as a login provider for your WebID.
```

Comment:
I'm wondering if this is strong enough? This text will only appear if they choose to register (send us a password) with their own WebID. But that password will be useless until they add that triple.